controls/listview/listview-features-load-on-demand.md

```diff
@@ -6,121 +6,118 @@ position: 8
 slug: listview-features-load-on-demand
 ---
 
-# Load On Demand
+# Load on Demand
 
-Loading a large data set on a mobile device has its challenges. One of the most popular approaches is using incremental data loading as the items needs to be visualized. **RadListView** is capable of doing this using its load on demand functionality. 
+Loading a large data set on a mobile device has its challenges. One of the most popular approaches is using incremental data loading when the items need to be visualized. **RadListView** does this by using its load-on-demand functionality.
 
 ## Configuration
 
-The following properties are related to the **LoadOnDemand** feature:
+The following properties control the **LoadOnDemand** feature:
 
-* **IsLoadOnDemandEnabled**(*bool*) - used to enable the load on demand mechanism of RadListView;
-* **IsLoadOnDemandActive** (*bool*) - this flag should be used when the data is retrieved asynchronously. It should be set to "True" when an async operation is started, to notify the ListView to show loading indicator and to "False" when items are loaded. 
```

Comment:
The definition is confusing because of the excessive use of passive voice and the _should_ modal verb.

https://testdocs.telerik.com/style-guide/basic-rules-and-guidelines#use-the-active-voice
https://testdocs.telerik.com/style-guide/basic-rules-and-guidelines#use-properly-should-could-must-have-to-need-to